terraform/db.tf

```diff
@@ -0,0 +1,39 @@
+
+resource "aws_db_subnet_group" "pgsql" {
+  name       = "pgsql"
+  subnet_ids = [aws_subnet.private_subnet.id]
+}
+
+resource "aws_rds_cluster_instance" "pgsql" {
+  count              = var.target_env == "prod" ? 2 : 1
+  identifier         = "${local.db_name}-${count.index}"
+  cluster_identifier = aws_rds_cluster.pgsql.id
+  instance_class     = "db.r5.large"
```

Comment:
This seems pretty big. How many records are we expecting?

Reply:
This code is copied from ltx and ets ..... Almost or a little bigger than HCAP.

Reply:
Maybe something smaller would do? I highly doubt we need R5 with 16 gb memory. 

Maybe use a general purpose category? https://aws.amazon.com/rds/instance-types/